tutorial/00_introduction_to_python.ipynb

```diff
@@ -7,9 +7,43 @@
    "source": [
     "# Introduction to Python\n",
     "\n",
-    "Let's execute our first Python command. Click on the bottom section and click on \"Run\" button on the top\n",
+    "Python is one of the world’s most used and most popular programming languages. It’s powerful, versatile, and easy to learn. Python is widely used in various applications, some notable ones:\n",
     "\n",
-    "TODO Organizers: give short introduction\n"
+    "- Web development\n",
+    "- Data Science\n",
+    "- Data analysis\n",
+    "- Machine learning\n",
+    "- Artificial Intelligence (AI)\n",
+    "- Scripting and tooling\n",
+    "\n",
+    "Many people say that **Python comes with batteries included.** It’s a fun way to state that it includes a comprehensive base library. In addition, because so many people use Python, hundreds of thousands of high-quality libraries and frameworks exist to get things done quickly and without hassle. You can do a lot with a little bit of Python code!\n",
+    "\n",
+    "Learning Python is a no-brainer, and I promise you will be up and running quickly with this Python tutorial. Regardless of your future in IT, it will be a helpful tool to have in your toolbox!\n",
```

Comment:
Nicely explained! Shall we use "we" instead of "I" throughout the course? So, it is coming from the ING family :)  `ex. we promise you`

Reply:
Good point, will do!